recipe/meta.yaml

```diff
@@ -38,16 +38,16 @@ requirements:
   host:
     - libblas
     - liblapack
-    - libgomp                      # [linux]
+    - libgomp                      # [not osx]
     - llvm-openmp                  # [osx]
```

Comment:
Would move both of these to `requirements/build` and drop them from other `requirements` sections as both have `run_exports/strong`, which automatically adds them to the other sections